### PR TITLE
scripts.d/s-adao: Add ng mode

### PR DESCRIPTION
**Pull request by estreby**
hello ! I just try to add ng mode for adao.
It may not be complete

### CODE REVIEW

File: scripts.d/s-adao [hunk 1]
```diff
@@ -72,7 +85,8 @@ function s-adao-config()
     local CDEBUG=$(is-enabled cmake-debug)
     local DISDOC=$(is-enabled no-doc)
 
-    cmake -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    cmake -DADAO_BUILD_DOC=OFF \
```
Comment (TurboGit) [inline scripts.d/s-adao:88]: this is not needed ad you have below:

`${DISDOC:+"-DADAO_BUILD_DOC=OFF"} \`

**Review by TurboGit — CHANGES_REQUESTED** [review]
Some changes are needed.

File: scripts.d/s-adao [hunk 2]
```diff
@@ -26,19 +26,32 @@ function s-adao-tar()
...
+function s-adao-common-build-depends()
+{
+    echo s-cmake
+}
+
 function s-adao-build-depends()
 {
     local DISDOC=$(is-enabled no-doc)
 
-    echo s-cmake
```
[the remark this answers is not in the document]
Comment (estreby) [reply scripts.d/s-adao]: Thank you it's pushed

File: scripts.d/s-adao (hunk 1 above)
Comment (TurboGit) [reply scripts.d/s-adao:84]: You forgot this one :)

**Review by TurboGit — APPROVED** [review]
Simpler and perfect. Thanks. Merging manually to fix conflicts due to my other changes.

**Pull request closed without merging**